CONTRIBUTING.md

```diff
@@ -8,6 +8,18 @@
 
 - (Recommended) If you use [Visual Studio Code], install recommended extensions to improve your development experience.
 
+## Contributing to Issues
+
+If you'd like to contribute to the Azure SDK for Rust, we recommend looking for issues with the following labels:
+
+- [design-discussion](https://github.com/Azure/azure-sdk-for-rust/issues?q=is%3Aopen+is%3Aissue+label%3Adesign-discussion) - These issues involve discussions about the design of features or APIs and are a great way to influence the future direction of the SDK.
+
+- [good first issue](https://github.com/Azure/azure-sdk-for-rust/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - These issues are a good starting point for new contributors, as they should be relatively straightforward to address.
```

Comment:
Improper grammar.
```suggestion
- [good first issue](https://github.com/Azure/azure-sdk-for-rust/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - These issues are a good starting point for new contributors since they should be relatively straightforward to address.
```